android/build.gradle

```diff
@@ -18,16 +18,24 @@ buildscript {
 }
 
 ext.getConfig = { String name ->
-    def env = System.getenv(name)
-    if (env != null) {
-        return env
+    File localPropertiesFile = project.file('local.properties')
+    if (localPropertiesFile?.exists()) {
+        Properties localProperties = new Properties()
+        localProperties.load(localPropertiesFile.newDataInputStream())
+        if (localProperties.containsKey(name)) {
+            return localProperties.getProperty(name)
+        }
+    }
+    if (project.hasProperty(name)) {
+        return project.getProperty(name)
     }
     def prop = System.getProperty(name)
     if (prop != null) {
         return prop
     }
-    if (project.hasProperty(name)) {
-        return project.getProperty(name)
+    def env = System.getenv(name)
```

Comment:
I think env variable should be checked first as it was before. The CI jobs tries to override the project properties using env variables. If it is checked last then always the project variables will be used those committed in related code base in GitHub repository.

Reply:
I converted the PR to Draft until we decide [this](https://github.com/nevissecurity/nevis-mobile-authentication-sdk-example-app-android/pull/11#discussion_r1439293887).